message/html/compose.js

```diff
@@ -64,6 +75,12 @@ exports.create = function (api) {
     })
     textArea.publish = publish // TODO: fix - clunky api for the keyboard shortcut to target
 
+    // load draft
+    let draft = api.drafts.sync.get(resolve(meta).root || '/public')
+    if (typeof draft === 'string') {
+      textArea.value = draft
+    }
+
```

Comment:
:fire: - We need to clear drafts on publish!

The perfect place is in the `done` callback, which is inside the `publish` function near the bottom